CONTRIBUTING.md

```diff
@@ -12,9 +12,9 @@ Be sure to include as much information including screenshots, text output, and b
 
 ## How to Request Enhancements
 
-First, please refer to the applicable [GitHub repository][github-repo] and search [the repository's GitHub issues][issues-list] to make sure your idea has not been (or is not still) considered.
+Firstly please refer to the applicable GitHub repository [github-repo] and search the repository's GitHub issues [issues-list] to make sure that your idea has not been (or is not still) considered.
```

Comment:
Github repository is mentioned twice and issues is also written twice.
You can revert these changes